packages/core/src/codewhisperer/views/lineAnnotationController.ts

```diff
@@ -201,16 +203,15 @@ export class EndState implements AnnotationState {
 }
 
 export class InlineChatState implements AnnotationState {
-    static static = 'amazonq_annotation_inline_chat'
-    id = InlineChatState.static
+    static id = 'amazonq_annotation_inline_chat'
+    id = InlineChatState.id
```

Comment:
The old code assigned a new variable. Now it looks like `id` is immediately reassigned (or is it an instance variable)? What is the intention here?